src/transformState.js

```diff
@@ -22,9 +22,6 @@ function transformState(state, actions) {
           delete state[key];
         }
         break;
-
-      default:
```

Comment:
it would be best if you had `defautl` case, where you throw some error - so user can know, that he gave you a wrong action type

Reply:
okay